vars/tmsUpload.groovy

```diff
@@ -94,7 +94,11 @@ void call(Map parameters = [:]) {
 
         def namedUser = jenkinsUtils.getJobStartedByUserId()
 
-        if (config.useGoStep == true) {
+        def useGoStep = config.useGoStep
+        if (useGoStep == null) {
+            useGoStep = true
+        }
+        if (useGoStep == true) {
```

Comment:
```suggestion
        if (useGoStep) {
```
This is true also if `useGoStep` is any string, such as "false". So the previous way is more explicit. Your choice.

Reply:
Good, thanks! Then it should be this way [1e8691e](https://github.com/SAP/jenkins-library/pull/4535/commits/1e8691e7843d06b680159d7716c38e8c70ff7144), i.e. the Groovy implementation will be used if and only if the value is false, otherwise the default goes for Golang implementation.